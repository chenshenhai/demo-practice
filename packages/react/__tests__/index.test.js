import App from '../src/app';

import React from 'react';
import renderer from 'react-test-renderer';

test('Test React ', () => {
  const component = renderer.create(
    <App beginNum={100} />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});