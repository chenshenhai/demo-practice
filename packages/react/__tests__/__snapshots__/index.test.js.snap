// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test React  1`] = `
<div>
  <div>
    count:
    100
  </div>
  <button
    onClick={[Function]}
  >
    Add
  </button>
</div>
`;
